### PR TITLE
add project urls such as changelog and docs

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,10 @@
+[metadata]
+project_urls =
+    Documentation=https://pythonhosted.org/watchdog/
```
Comment (BoboTiG) [inline setup.cfg:3]:
```suggestion
    Documentation=https://python-watchdog.readthedocs.io/en/stable/
```

**Comment by BoboTiG** [comment]
Thanks @graingert  :)

File: setup.cfg [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 project_urls =
-    Documentation=https://pythonhosted.org/watchdog/
```
Comment (graingert) [inline setup.cfg]: @BoboTiG I grabbed this from the github description, so you should update it there

Comment (BoboTiG) [reply setup.cfg]: Yes, I am working on it :)

**Pull request merged**